src/Value/CalcFunction.php

```diff
@@ -59,10 +59,10 @@ public static function parse(ParserState $parserState, bool $ignoreCase = false)
             } else {
                 if (\in_array($parserState->peek(), $operators, true)) {
                     if (($parserState->comes('-') || $parserState->comes('+'))) {
+                        $nextChar = $parserState->peek(1, 1);
                         if (
                             $parserState->peek(1, -1) !== ' '
```

Comment:
As part of this change, could we also allow any type of whitespace before the operator?